### PR TITLE
[RFR] Responsive UI

### PR DESCRIPTION
**Pull request by fzaninotto**
Make the Mobile interface usable.

responsive

- [x] Sidebar
- [x] AppBar
- [x] Datagrid (introducing `<SimpleList>` and `<Responsive>` components)
- [x] Actions
- [x] Pagination

### CODE REVIEW

**Comment by fzaninotto** [comment]
Switching to RFR

**Comment by JulienDemangeon** [comment]
I do not understand the "const isMobile = width === 1;" everywhere ?

File: src/mui/detail/Show.js [hunk 1]
```diff
@@ -54,21 +58,25 @@ export class Show extends Component {
...
-                {data && React.cloneElement(children, {
-                    resource,
-                    basePath,
-                    record: data,
-                })}
-            </Card>
+            <div>
+                {isMobile && <AppBar title={titleElement} />}
+                <Card style={{ opacity: isLoading ? 0.8 : 1 }}>
+                    <CardActions style={{ zIndex: 2, display: 'inline-block', float: 'right' }}>
```
Comment (djhi) [inline src/mui/detail/Show.js]: styles should be in a separate object

**Comment by djhi** [comment]
@JulienDemangeon It's a material ui constant

**Pull request merged**